qt/scientific_interfaces/Indirect/Elwin.cpp

```diff
@@ -19,8 +19,18 @@ using namespace MantidQt::API;
 
 namespace {
 Mantid::Kernel::Logger g_log("Elwin");
+
+MatrixWorkspace_sptr getADSMatrixWorkspace(std::string const &workspaceName) {
+  return AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(
+      workspaceName);
+}
+
+bool isWorkspacePlottable(MatrixWorkspace_sptr workspace) {
+  return workspace->blocksize() > 1 ? true : false;
```

Comment:
It is enough to `return workspace->blocksize() > 1;`

By the way, `blocksize` can be a costly operation (please check). It may be advantageous to `return workspace->y(0).size() > 1;`.